README.md

```diff
@@ -94,6 +94,22 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 
 ## Deployment
 
+### Firebase App Hosting
+
+ 1. [Fork this Repository](https://github.com/google-gemini/gemini-image-editing-nextjs-quickstart/fork)
+ 2. Create a Firebase Project and upgrade it to the [Blaze Plan](https://firebase.google.com/docs/app-hosting/get-started#you-begin).
+ 3. Create a Firebase App Hosting Backend in [the Firebase Console](https://console.firebase.google.com/project/_/apphosting) or with the [Firebase CLI](https://firebase.google.com/docs/cli#setup_update_cli):
```

Comment:
https://console.firebase.google.com/project/_/apphosting

This URL looks like it is going to error out

Reply:
The URL takes you to the Firebase project selection page and redirects you the App Hosting tab after selecting a project.